packages/p2p/src/components/__tests__/app-content.spec.js

```diff
@@ -33,6 +33,18 @@ jest.mock('Components/orders/orders', () => jest.fn(() => 'Orders'));
 jest.mock('Components/buy-sell/buy-sell', () => jest.fn(() => 'BuySell'));
 jest.mock('Components/my-profile', () => jest.fn(() => 'MyProfile'));
 
+jest.mock('@sendbird/chat', () => ({
```

Comment:
do we need to mock sendbird in components outside orders page? cause i dont think these components use sendbird

Reply:
@farhan-nurzi-deriv, This component uses  `general_store`, which uses `sendbird_store`. `sendbird_store` uses `SendbirdChat`, so we need to `mock` it here as well.